src/Sulu/Bundle/WebsiteBundle/Controller/DefaultController.php

```diff
@@ -81,4 +81,15 @@ public function redirectAction(Request $request)
             $request->query->all()
         );
     }
+
+    public function structureNotFoundAction(StructureInterface $structure, $preview = false, $partial = false)
+    {
+        $structureFactory = $this->get('sulu_content.structure.factory');
+        $metadata = $structure->getStructure();
+
+        throw new \InvalidArgumentException(sprintf(
```

Comment:
Yeah, this is just for an example.

On Thu, Mar 17, 2016 at 07:06:32AM -0700, Wachter Johannes wrote:
>    In [1]src/Sulu/Bundle/WebsiteBundle/Controller/DefaultController.php:
> 
>  > @@ -81,4 +81,15 @@ public function redirectAction(Request $request)
>  >              $request->query->all()
>  >          );
>  >      }
>  > +
>  > +    public function structureNotFoundAction(StructureInterface $structure, $preview = false, $partial = false)
>  > +    {
>  > +        $structureFactory = $this->get('sulu_content.structure.factory');
>  > +        $metadata = $structure->getStructure();
>  > +
>  > +        throw new \InvalidArgumentException(sprintf(
> 
>    should at least return 404 google understands no fun for 500 on a page.
> 
>    —
>    You are receiving this because you authored the thread.
>    Reply to this email directly or [2]view it on GitHub
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu-io/sulu/pull/2139#discussion_r56508558
>    2. https://github.com/sulu-io/sulu/pull/2139/files/cd833019fdc23d5257f77e15cbfa223d24841ce5#r56508558
>    3. https://github.com/sulu-io/sulu/pull/2139/files/cd833019fdc23d5257f77e15cbfa223d24841ce5#r56508558